plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -8510,6 +8512,113 @@ public void testCreateAcidTableUnsupported()
         assertQueryFails("CREATE TABLE acid_unsupported WITH (transactional = true) AS SELECT 123 x", "FileHiveMetastore does not support ACID tables");
     }
 
+    @Test
+    public void testExtraProperties()
+    {
+        String tableName = format("%s.%s.test_extra_properties", getSession().getCatalog().get(), getSession().getSchema().get());
+        @Language("SQL") String createTableSql = format("""
+                        CREATE TABLE %s (
+                          c1 integer)
+                        WITH (
+                          extra_properties = MAP(ARRAY['extra.property'], ARRAY['true']),
+                          format = 'ORC'
+                        )""",
+                tableName);
+        MaterializedResultWithQueryId result = getDistributedQueryRunner().executeWithQueryId(getSession(), createTableSql);
+        QueryId queryId = result.getQueryId();
+        String nodeVersion = (String) computeScalar("SELECT node_version FROM system.runtime.nodes WHERE coordinator");
+        assertQuery(
+                "SELECT * FROM \"test_extra_properties$properties\"",
+                "SELECT 'workaround for potential lack of HIVE-12730', 'false', 'true', '0', '0', '" + queryId + "', '" + nodeVersion + "', '0', '0', 'false'");
+        MaterializedResult actualResult = computeActual("SHOW CREATE TABLE " + tableName);
+        String expectedShowCreateTableSql = "CREATE TABLE hive.tpch.test_extra_properties (\n" +
+                "   c1 integer\n" +
+                ")\n" +
+                "WITH (\n" +
+                "   extra_properties = map_from_entries(ARRAY[ROW('extra.property', 'true')]),\n" +
+                "   format = 'ORC'\n" +
+                ")";
+        assertEquals(getOnlyElement(actualResult.getOnlyColumnAsSet()), expectedShowCreateTableSql);
+        assertUpdate("DROP TABLE " + tableName);
+    }
+
```

Comment:
```
trino:default> create table test10( 
            ->    name varchar(10), 
            ->    comment varchar(10), 
            ->    nationkey bigint, 
            ->    regionkey bigint, 
            ->    short_name varchar(100) 
            ->      with (partition_projection_type = 'enum', partition_projection_values = array['PL1', 'CZ1'])
            -> )
            -> with (
            ->     partitioned_by = array['short_name'], 
            ->     partition_projection_enabled = true, 
            ->     extra_properties = map_from_entries(ARRAY[ROW('extra2', 'hello'),ROW('extra1', 'world')]));
CREATE TABLE
trino:default> show create table test10;
                                                                                      Create Table                                                                                      
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 CREATE TABLE hive.default.test10 (                                                                                                                                                     
    name varchar(10),                                                                                                                                                                   
    comment varchar(10),                                                                                                                                                                
    nationkey bigint,                                                                                                                                                                   
    regionkey bigint,                                                                                                                                                                   
    short_name varchar(100)                                                                                                                                                             
 )                                                                                                                                                                                      
 WITH (                                                                                                                                                                                 
    extra_properties = map_from_entries(ARRAY[ROW('extra2', 'hello'),ROW('projection.short_name.type', 'enum'),ROW('extra1', 'world'),ROW('projection.short_name.values', 'PL1,CZ1')]), 
    format = 'ORC',                                                                                                                                                                     
    partition_projection_enabled = true,                                                                                                                                                
    partitioned_by = ARRAY['short_name']                                                                                                                                                
 )                                                                                                                                                                                      
(1 row)
```

Is this intended to be displayed this way ?

cc @aczajkowski

Reply:
I would be very cautious here as it was a conscious decision to not blindly present all properties as table properties.
As in case of partition projection in Hive some properties are presented as table properties even they are pointing to columns. Which creates a lot of potential issues (e.g dropping column, keeping property etc..). 
Thats why we decided to keep them / present as table column properties.
So seems there should be no problem, as they will be just additionally presented in `extra_properties` but it might bring confusion here as they will be named differently as well. 
CC: @findepi

Reply:
> So seems there should be no problem, as they will be just additionally presented in `extra_properties`

they shouldn't be double-presented. if these show up as column properties, they shouldn't show up as table properties.

Reply:
Is fixing  #16873 something that you're considering doing @Praveen2112  ? 
I'm thinking it is worth taking a stab at it just to get more knowledge of eventual corner cases which are not yet easily visible and are relevant for this feature.